automation/taskcluster/lib/tasks.py

```diff
@@ -104,6 +98,43 @@ def beetmover_task(self, name, description, version, artifact_id,
             }
         }
 
+    def build_push_task(self, signing_task_id, name, description, commit, apks=[], scopes=[]):
+        created = datetime.datetime.now()
+        expires = taskcluster.fromNow('1 year')
+        deadline = taskcluster.fromNow('1 day')
+
+        return {
+            "workerType": 'mobile-pushapk-v1',
+            "taskGroupId": self.task_id,
+            "schedulerId": self.scheduler_id,
+            "expires": taskcluster.stringDate(expires),
+            "retries": 5,
+            "created": taskcluster.stringDate(created),
+            "tags": {},
+            "priority": 'lowest',
+            "deadline": taskcluster.stringDate(deadline),
+            "dependencies": [self.task_id, signing_task_id],
+            "routes": [],
+            "scopes": scopes,
+            "requires": 'all-completed',
+            "payload": {
+                "commit": commit,
+                "google_play_track": 'nightly',   # TODO @jlorenzo nightly Google Play
```

Comment:
I confirm this is the expected track. @pocmo, releng doesn't (and shouldn't) have rights on publishing the APKs. Who will be the owner of the app? This person must create the track thanks to these instructions https://support.google.com/googleplay/android-developer/answer/3131213#runtest

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1508294